web_api/__init__.py

```
@@ -0,0 +1,319 @@
+        description = str(payload.get("description") or "").strip()
+        config = payload.get("config")
+        if not isinstance(config, dict) or not config:
+            if payload.get("from_global", False):
+                config = self.profile_store.template_from_global(self.config_manager)
+            else:
+                config = {}
+        template = self.profile_store.create_template(name, description, config)
+        return _ok(template, "模板已创建")
+
+    async def update_profile(self):
+        """更新模板（名称/描述/配置）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)